test/jdk/sun/security/pkcs11/PKCS11Test.java

```diff
@@ -700,10 +700,10 @@ private static Map<String, String[]> getOsMap() {
         osMap.put("Linux-arm-32", new String[] {
                 "/usr/lib/arm-linux-gnueabi/nss/",
                 "/usr/lib/arm-linux-gnueabihf/nss/" });
-        osMap.put("Linux-aarch64-64", new String[] {
-                "/usr/lib/aarch64-linux-gnu/",
-                "/usr/lib/aarch64-linux-gnu/nss/",
-                "/usr/lib64/" });
+//        osMap.put("Linux-aarch64-64", new String[] {
```

Comment:
Perhaps add a comment above this noting why these are being excluded. Could include the bugid for now until it is fixed.